easyship-doorkeeper-jwt.gemspec

```
@@ -18,9 +18,9 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'doorkeeper', '~> 3.1'
-  spec.add_dependency 'jwt', '~> 1.5', '>= 1.5.4'
+  spec.add_dependency 'doorkeeper', '~> 5.2'
```

It's 5.1 instead of 5.2
```suggestion
  spec.add_dependency 'doorkeeper', '~> 5.1'
```